modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -443,6 +446,8 @@ def groupby_agg(self, by, axis, agg, groupby_args, **kwargs):
         groupby_cols = by_frame.columns
         if isinstance(agg, dict):
             agg_cols = agg.keys()
+        elif selection is not None:
```

Comment:
Let's add some tests for omnisci engine.

Reply:
added